sql-plugin/src/main/320+/scala/org/apache/spark/sql/execution/datasources/rapids/DataSourceStrategyUtils.scala

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright (c) 2021-2022, NVIDIA CORPORATION.
```

Comment:
accidentally removed 2021?

Reply:
This is a case of git mistakenly thinking this is a file rename.  The files have very little in common other than a few imports and the ASL2.0 license boilerplate.  Otherwise the two files are not related.